### DISCUSSION TITLE
Buttons "Next file" , "Previous file"

### DISCUSSION THREAD

**Discussion by Valeriys2013** [WinMerge/winmerge#736] [Ideas]
I think it would be convenient if buttons "Next file" , "Previous file" navigate to next or previous file WITH DIFFERENCES.
At least as an optional regime

**Comment by sdottaka** [comment]
In version 2.16.10, the Next File and
 Previous File buttons have been added to the toolbar.
(Note: These buttons are only enabled when comparing files from the folder comparison window)

![image](https://user-images.githubusercontent.com/98126/133999294-8aea73ce-cf09-4b85-92a4-fee8dee769c2.png)
